bot.py

```diff
@@ -174,20 +174,24 @@ async def daily(interaction: discord.Interaction):
 
 
 @tree.command(guild=guild)
-async def balance(interaction: discord.Interaction):
-    """Get your balance."""
-    resp = hyperion_session.get(
-        f"{config.hyperion_endpoint}/api/v1/accounts/{interaction.user.id}"
-    )
+async def balance(interaction: discord.Interaction, user: Optional[discord.user]):
+    """Get balance of a user."""
+
+    if user is None:
+        user = interaction.user
+
+    resp = hyperion_session.get(f"{config.hyperion_endpoint}/api/v1/accounts/{user.id}")
     if resp.status_code != 200:
         await interaction.response.send_message(
             f"Error getting account.\n```\n{resp.json()['detail']}\n```",
         )
         return
 
+    prefix = "You" if user == interaction.user else f"{user.name}'s"
+
     account = Account(**resp.json())
     await interaction.response.send_message(
-        f"You have {account.balance} {currency.plural_form}."
+        f"{prefix} {account.balance} {currency.plural_form}."
```

Comment:
This will currently give a format of something like
> You 20 Gumplets.

or

> Dan's 20 Gumplets.

Reply:
maintainer commits welcome